### DISCUSSION TITLE
Would make sense to add support for tansport using NATS micro ?

### DISCUSSION THREAD

**Discussion by mcosta74** [go-kit/kit#1284] [Ideas]
NATS micro has been introduced since a while and it adds some useful features to monitor and manage micro-services connected to NATS.

Would you consider to add a new transport that supports it? I can work on it and raise a PR but I'd like to get first consesus if such feature will be accepted or not

**Comment by peterbourgon** [comment]
You're free to do so, but there's no reason it needs to be included in the main Go kit repo, it should work fine in your own. I'll happily add a link to the README.

**Reply by mcosta74** [reply]
ok, I'll create it as "go-kit" extension and I'll publish in separate repo
